pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -1445,7 +1445,88 @@ func (p *Provider) Create(
 			return err
 		}
 	}
-	return propagateDiskLabels(l, project, labels, zoneToHostNames, &opts)
+	return propagateDiskLabels(l, project, labels, zoneToHostNames, opts.SSDOpts.UseLocalSSD)
+}
+
+// computeGrowDistribution computes the distribution of new nodes across the
+// existing instance groups. The distribution is computed naively, for
```

Comment:
Just curious, what would be a less naive way of distribution? Avoiding all the nodes in a cluster getting placed in the same instance group?